src/plone/volto/locales/de/LC_MESSAGES/plone.volto.po

```diff
@@ -38,8 +38,8 @@ msgid "Creates a default page for the site using slate blocks"
 msgstr ""
 
 #: plone/volto/behaviors/configure.zcml:42
-msgid "Head title field"
-msgstr "Kopftitel Feld"
```

Comment:
I think we should keep the existing translations for "head title"; it's better than removing the translations.

Reply:
I think that it is automatically updated once we removed the text. Translation is automatically deleted.